go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kiegroup/kogito-cloud-operator
 
 require (
+	github.com/PaesslerAG/jsonpath v0.1.1
```

Comment:
I added this golang library which is quite useful to parse expressions in JSON. However, not sure if we are allowed to add libraries or if you already know about any other for this. 
Morever, I've added a section in our README file to explain a bit further this.
What do you think?

Reply:
My only concern is with their license and contribution level. Have you done a research regarding the best/most active library out there? This one has only one contributor and their license is BSD. Ideally we should rely on Apache.

Reply:
After the latest refactor, the step that made use of this library is no longer used. I will remove this step and this dependency.